### PR TITLE
Added promo card Interplanetary Trade (#539)

### PR DESCRIPTION
**Pull request by gomfunkel**
Includes pull requests #569, #570, #571, #572 and #573, please merge those first.

### CODE REVIEW

File: src/cards/promo/InterplanetaryTrade.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { IProjectCard } from '../IProjectCard';
```
Comment (alrusdi) [inline src/cards/promo/InterplanetaryTrade.ts:1]: Double quotes

File: src/cards/promo/InterplanetaryTrade.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            player.corporationCard.tags.forEach((tag) => allTags.push(tag));
+        }
+        player.playedCards.filter((card) => card.cardType !== CardType.EVENT).forEach((card) => {
+            card.tags.forEach((tag) => { 
+                allTags.push(tag);
+            });
+        });
+        for (const tags of allTags) {
+            if (tags === Tags.WILDCARD) {
+                wildcardCount++;
```
Comment (alrusdi) [inline src/cards/promo/InterplanetaryTrade.ts:29]: I think you should calc all existing tags and increment this only if there exists a unused tags.
I mean, wild tag is not the tag itself - it is a copy of another tag.
Quote form internets:
"The wild tag becomes another tag of your choice on your turn only. So if you already have all other tags, it is not a unique tag"

**Comment by gomfunkel** [comment]
Will be updated with change requests from #570, #571, #572 and #573 before merging.

File: src/cards/promo/InterplanetaryTrade.ts (hunk 1 above)
Comment (gomfunkel) [reply src/cards/promo/InterplanetaryTrade.ts]: Fixed.

File: src/cards/promo/InterplanetaryTrade.ts (hunk 2 above)
Comment (gomfunkel) [reply src/cards/promo/InterplanetaryTrade.ts:29]: Fixed and added additional test for it.

**Comment by gomfunkel** [comment]
@alrusdi Updated the pull request with (hopefully) all change requests from #570, #571, #572, #573 and this one. Further cards will be branched off directly from master for easier handling. Sorry again.

**Pull request merged**